lib/rouster.rb

```diff
@@ -718,14 +746,14 @@ def _run(command)
     output = File.read(tmp_file)
     File.delete(tmp_file) or raise InternalError.new(sprintf('unable to delete [%s]: %s', tmp_file, $!))
 
-    self.output.push(output)
+    self.ssh_stdout.push(output)
```

Comment:
think we need to take a different tact here:
  * create a different class variable to hold output from non-SSH runs OR
  * don't cache output of exitcodes/output for non-SSH runs and just return it from this method

in this case we can't/don't distinguish between STDERR and STDOUT, so i'd be fine if the class variable didn't either